articles/native-platforms/windowsphone/01-login.md

```diff
@@ -6,9 +6,10 @@ description: This tutorial will show you how to use the Auth0 Windows Phone SDK
 ## Windows Phone Tutorial
 
 <%= include('../../_includes/_package', {
-  pkgRepo: 'Auth0.WindowsPhone',
```

Comment:
@mehreencs87 Hi, another change was introduced and we need to apply it before this PR is merged. `pkgOrg` was added:
- if the sample is under `auth0-samples` repo add the line: `pkgOrg: 'auth0-samples'`
- if the sample is under `auth0` repo add the line: `pkgOrg: 'auth0'`
Details: https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects

Can you please update this? Thanks.